<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransfersWidget</class>
 <widget class="QWidget" name="TransfersWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="sWidget">
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>600</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="pNoTransfers">
      <property name="minimumSize">
       <size>
        <width>518</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>518</width>
        <height>16777215</height>
       </size>
      </property>
      <widget class="QPushButton" name="lNoTransfers">
       <property name="geometry">
        <rect>
         <x>6</x>
         <y>8</y>
         <width>518</width>
         <height>281</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>518</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>518</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;</string>
       </property>
       <property name="text">
        <string>NO TRANSFERS</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="pTransfers">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTreeView" name="tvTransfers">
         <property name="styleSheet">
          <string notr="true">background-color: white;
alternate-background-color: #D3D3D3;
paint-alternating-row-colors-for-empty-area: false;</string>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>true</bool>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
